### ISSUE TITLE
list-table :widths: ignored

### ISSUE THREAD

**Issue by rpuntaie** [jgm/pandoc#4260]
Pandoc 2.1

With pandoc I'm not able to fit this table onto the page width

```
Table example

.. list-table::
   :name:
   :widths: 17 7 7 7 7 7 7 7 34
   :header-rows: 1

   * - SLOT
     - 0
     - 1
     - 2
     - 3
     - 4
     - 5
     - 6
     - 7

   * - High
   
       time
       
       stimulation
       
       LED
     - 18
     
       dutycycle
       
       25%
     - 18
     
       dutycycle
       
       25%
     - 72
     
       dutycycle
       
       100%
     - 18
     
       dutycycle
       
       100%
     - 18
     
       dutycycle
       
       25%
     - 18
     
       dutycycle
       
       25%
     - 18
     
       dutycycle
       
       25%
     - 18
     
       dutycycle
       
       25%
```

rst2html.py considers `:widths:`
The table can be fitted on the page.

**Comment by mb21** [comment]
@rpuntaie you should really get into the habit of including the whole command (in addition to the minimal input) in your bug reports.

Your input on `pandoc -f rst -t native` gives:

    Table [] [AlignDefault, ...] [0.17,7.0e-2,7.0e-2,7.0e-2,7.0e-2,7.0e-2,7.0e-2,7.0e-2,0.34] ...

so the problem might be in the writer?

**Comment by rpuntaie** [comment]
It is with latex writer, then.

My commands:

```
pandoc -f rst -t latex -V geometry:margin=2.5cm -V papersize:a4 tmp.txt -o tmp.latex
pandoc -f rst -V geometry:margin=2.5cm -V papersize:a4 tmp.txt -o tmp.pdf
```

**Comment by jgm** [comment]
Here's what I get with the current dev version:

<img width="975" alt="screen shot 2018-01-15 at 10 21 47 am" src="https://user-images.githubusercontent.com/3044/34956445-f5078b8e-f9dd-11e7-9d54-41239cabfb3a.png">

The column widths are correct, though the table looks bad because the word "dutycycle" is too wide to fit in such narrow cells.

If you get something different, it may be that the issue has been fixed by commit c5ba3b8ee32704f1fdf241480ba64950f8ceedd8.

**Comment by jgm** [comment]
I confirm that the released pandoc 2.1 does not give the right results.  So, this bug has been fixed by [the commit mentioned above](https://github.com/jgm/pandoc/commit/c5ba3b8ee32704f1fdf241480ba64950f8ceedd8).
